geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/index/id/legacy/IdIndexV3.scala

```diff
@@ -36,17 +36,9 @@ object IdIndexV3 {
 
   class IdIndexKeySpaceV3(sft: SimpleFeatureType, override val sharing: Array[Byte]) extends IdIndexKeySpace(sft) {
 
-    private val rangePrefixes = {
```

Comment:
shards are not used by the id index, so this block was redundant as sharding.length will always be 0